test/common_utils/backend_utils.py

```diff
@@ -29,8 +29,8 @@ def supports_mp3(backend):
 def set_audio_backend(backend):
     """Allow additional backend value, 'default'"""
     if backend == 'default':
-        if 'sox' in BACKENDS:
-            be = 'sox'
```

Comment:
note: this is removing "sox" from "set_audio_backend"

Reply:
no, it's changing the default backend to be `sox_io` when it's available.